script.js

```diff
@@ -29,4 +29,212 @@ aboutLink.addEventListener('click', () => {
 contactLink.addEventListener('click', () => {
   mobileMenu.classList.replace('show_menu', 'hide_menu');
   header.classList.replace('hide_menu', 'show_menu');
-});
\ No newline at end of file
+});
+
+// Array of project objects to load the content dynamically
+const works = [
+  {
+    id: 'featureWork',
+    name: 'Multi-Post Stories',
+    description: `A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard dummy text
+                  ever since the 1500s, when an unknown printer took a standard
+                  dummy text.`,
+    featImage: 'imgs/img-placeholder-fw.png',
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: 'https://luigirazum.github.io/',
+    sourceLink: 'https://github.com/luigirazum/mv-portfolio-project',
+  },
+  {
+    id: 'work1',
+    name: 'Profesional Art Printing Data',
+    description: `A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard.`,
+    featImage: '../imgs/placeholder-img.png',
```

Comment:
[OPTIONAL] Please correct the featured image path in your array, it should be `./imgs/placeholder-img.png` I think you have two dots before and that is why images are not showing up on the page.